src/if9/authenticate.spec.ts

```diff
@@ -115,6 +113,9 @@ describe("TEST validateServiceRequest", () => {
     const req = new MockExpressRequest(rdlOptions);
     const res = new MockExpressResponse();
 
+    const spyVIN = jest.spyOn(clients, "isDeviceRegistered");
+    spyVIN.mockReturnValueOnce(true);
+
     authenticate.validateServiceRequest(req, res, next);
     expect(next).toHaveBeenCalled();
   });
```

Comment:
I removed the `initDeviceDB` method from [src/ifop/clients.ts](https://github.com/wzr1337/if9simu/pull/11/commits/a3483b0854eef59b53673d5c3ddf1bc6b2d9fe46#diff-3bbb73a5164b0493187165f4dfdc30af) and instead mocked `isDeviceRegistered` to test the function properly.